app/presenters/content_item_presenter.rb

```diff
@@ -52,4 +52,8 @@ def links
   def present_linked_item(linked_item)
     LinkedItemPresenter.new(linked_item, @api_url_method).present
   end
+
+  def document_type_expanded?
+    @item.document_type =~ /travel_advice/
```

Comment:
`attr_reader :item, :api_url_method` would avoid accessing the instance variables directly